libnavigation-ui/src/main/java/com/mapbox/navigation/ui/summary/SummaryModel.java

```diff
@@ -23,9 +24,10 @@
 
   public SummaryModel(final Context context, final DistanceFormatter distanceFormatter, final RouteProgress progress,
                       final @TimeFormat.Type int timeFormatType) {
-    final Locale locale = progress.getRoute() == null ? null :
+    final Locale locale = progress.getRoute().equals(DirectionsRoute.builder().build()) ? null :
```

Comment:
nit: could introduce an "isEmpty()". cache the empty, also a little nicer abstraction

Reply:
From internal discussion @kmadsen 

> would need to add it to mapbox-java
> ```kotlin
> val emptyDirectionsRoute: private static final DirectionsRoute emptyDirectionsRoute = DirectionsRoute.builder().build();
> public boolean isEmpty(): Boolean {
>     return this.equals(emptyDirectionsRoute
> }
>```
>
> or a RouteProgress extension function
> ```kotlin
> private val emptyDirectionsRoute: DirectionsRoute = DirectionsRoute.builder().build()
> fun DirectionsRoute.isEmpty(): Boolean {
>     return this == emptyDirectionsRoute
> }
> ```